spec/spec.md

```diff
@@ -506,39 +506,39 @@ In the high level, an overall endpoint-to-endpoint TSP routed mode involves thre
 
-For a common case of `k = 1 or 2`, the route hop list MAY be acquired in a simple arrangement:
-- The source endpoint `A` has chosen an intermediary `P` and established a relationship `(VID_a1, VID_p1)`, then `VID_hop1` is `VID_p1`. This VID is used as the `VID_rcvr` in the envelope.
-- The destination endpoint `B` has chosen an intermediary `Q` and established a relations `(VID_b1, VID_q1)`, then `VID_exit` is `VID_q1`. The intermediary `Q`, as a common service provider, may have published a well-known public `VID_q0`, then `VID_hop2` is `VID_q0`.
-- The destination endpoint `B` MAY share the routing information `VID_q0, VID_b1` in the Out-Of-Band Introduction mechanism or control payload in another TSP relationship, together with its chosen `VID_destination`.
-- The source endpoint `A` combines them together to form the whole message: `[VID_a1, VID_p1, VID_q0, VID_b1, Payload]`.
-- If the intermediary chosen by `B` is also acceptable to `A`, and the parties accept a single intermediary (with its potential loss of some metadata protection), then the resulting route may simply be `[VID_sndr, VID_intermediary_rcvr, VID_exit, Payload]`.
+For the common case of `k = 1 or k = 2`, the route hop list MAY be acquired via a simple arrangement:
+- The source endpoint `A` chooses an intermediary `P` and establishs a relationship with `P`, `(VID_a1, VID_p1)`, then `VID_hop1` is `VID_p1`. This VID is used as the `VID_rcvr` in the envelope.
+- The destination endpoint `B` choses an intermediary `Q` and establishs a relationship with `Q`, `(VID_b1, VID_q1)`, then `VID_exit` is `VID_q1`. The intermediary `Q`, as a common service provider, may have published a well-known public `VID_q0`, then `VID_hop2` could be `VID_q0`.
+- The destination endpoint `B` MAY share the routing information `(VID_q0, VID_b1)` in the Out-Of-Band Introduction mechanism or via a control payload TSP message in another TSP relationship, together with its chosen `VID_destination`.
+- The source endpoint `A` combines the routes together to form the whole message: `[VID_a1, VID_p1, VID_q0, VID_b1, Payload]`.
+- If the intermediary chosen by `B` is also acceptable to `A`, and the parties accept a single intermediary (with the potential loss of some metadata protection), then the resulting route may simply be `[VID_sndr, VID_intermediary_rcvr, VID_exit, Payload]`.
 
-TSP routed messages have the same TSP Envelope but extends the header fields of the payload with the following structure:
+TSP routed messages have the same TSP Envelope as TSP messages sent in direct mode but extend the header field of the payload with the following structure:
 
 ``` text
-Payload_Header_Fields = {VID_sndr|NULL, VID_hop2, ..., VID_hopk, VID_exit}
+Payload_Header_Fields = {VID_sndr|NULL, VID_rcvr, VID_hop2, ..., VID_hopk, VID_exit}
```

Comment:
I wanted to point out that I made this change to match the paragraph and the convention I think this section is following.  Double check that I'm right as maybe I misunderstood.